packages/grid/_modules_/grid/components/panel/filterPanel/GridFilterForm.tsx

```diff
@@ -46,10 +59,24 @@ const GridFilterFormRoot = styled('div', {
   overridesResolver: (props, styles) => styles.filterForm,
```

Comment:
#3716 made me remember that if we add new classes we need to update the `overridesResolver`.